bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BArrayType.java

```diff
@@ -1,20 +1,20 @@
 /*
-*   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
-*
-*  WSO2 Inc. licenses this file to you under the Apache License,
-*  Version 2.0 (the "License"); you may not use this file except
-*  in compliance with the License.
-*  You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing,
-* software distributed under the License is distributed on an
-* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-* KIND, either express or implied.  See the License for the
-* specific language governing permissions and limitations
-* under the License.
-*/
+ *   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Licence header should be 2020

Reply:
Changed it in commit [52879ba](https://github.com/ballerina-platform/ballerina-lang/pull/22708/commits/52879ba833b656e946a3650bcd6a2ef5b9d0683e)

Reply:
We change license header only in new files? why is this requested here?

Reply:
I initially thought this was new file and just realize this is a just formatting. Yes correct. 2019 should be ok. Sorry for that. :)